.changeset/rude-walls-add.md

```diff
@@ -0,0 +1,6 @@
+---
+"@directus/api": major
```

Comment:
> Is this a breaking change? we're blocking edits on system fields however while you could do the edit it was not working as expected before.

IMO if it was not working to begin with it is not something someone can be dependant on and therefor not a breaking change.

Reply:
Previous versions allow for updating of `meta` and only `is_indexed` is allowed now. 


@br41nslug Does the updating of `meta` cause any issues? It previously creates a new record in `directus_fields`.

Reply:
As discussed it does, in the interest of being more cautious we will leave this as major.